src/main/java/core/basesyntax/interfaces/AreaCalculator.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.interfaces;
+
+public interface AreaCalculator {
+    double ZERO_POINT_FIVE = 0.5;
+    double TWO = 2.0;
+    double FOUR = 4.0;
```

Comment:
generally, it's bad practice to use interface constants. Here are some details: https://stackoverflow.com/questions/2659593/what-is-the-use-of-interface-constants